### PR TITLE
Blog posts and website page for announcing call for teaching workshop partnerships and UGM

### PR DESCRIPTION
**Pull request by jennaswa**

### CODE REVIEW

**Comment by jennaswa** [comment]
Still learning how to best work with forked repositories (think I need to start using branches...), so I ended up with two blog posts in this PR: one for asking for teaching partnerships and one for announcing the UGM.

File: _posts/2023-05-17-save-date-ugm2023.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: Save the Date - MDAnalysis UGM 2023
+---
+
+We are thrilled to announce the first ever MDAnalysis User Group Meeting (UGM)! The MDAnalysis UGM 2023 will take place September 27-29 (Wednesday-Friday) at the [University of Lisbon][uniL] (Portugal). To foster a connection opportunity for interdisciplinary researchers and developers across biomolecular simulations, soft matter and biophysics, this year's event will be held in-person and free to attend for all.
```
Comment (orbeckst) [inline _posts/2023-05-17-save-date-ugm2023.md:6]: Do we have to say here that it's free or can we provide details in a later announcement with registration details? That would give us more flexibility.

How are we handling registration? Will there be a wait list? How are we preventing no-shows from taking up seats?

File: _posts/2023-05-17-save-date-ugm2023.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: Save the Date - MDAnalysis UGM 2023
+---
+
```
Comment (orbeckst) [inline _posts/2023-05-17-save-date-ugm2023.md:5]: Do we have a nice image?

File: pages/ugm2023.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+* Community events, including a hackathon and social outings
+
+## Program and Keynotes
+A draft agenda and keynote speaker(s) will be announced soon.
+
+## Registration and Call for Proposals
+Registration and a call for proposals will open soon.
+
+## Pricing
+The event will be free to attend for all.
```
Comment (orbeckst) [inline pages/ugm2023.md:29]: See above... do you really want to make it free? With no "skin in the game" people may just register and never show up and take up seats.

**Comment by orbeckst** [comment]
> think I need to start using branches...

Yes, that's what I would have normally suggested when I saw two unrelated entries. But in the grand scheme of things it's no big deal.

File: _posts/2023-05-17-save-date-ugm2023.md (hunk 2 above)
Comment (jennaswa) [reply _posts/2023-05-17-save-date-ugm2023.md:10]: Added an image of Lisbon with an MDA banner logo. Can't seem to get rid of the large spacing between the image and start of the text though

File: _posts/2023-05-17-save-date-ugm2023.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+## What to Expect
+Whether you are a new MDAnalysis contributor, an MDAnalysis core developer, or a researcher using (or interested in using) MDAnalysis for your work in academia or industry, this UGM is for you. You can expect:
+* Keynote talks from experts in the molecular dynamics simulations space
+* Talks related to developing and using MDAnalysis for diverse scientific applications
+* Tutorials diving into new features
+* Direct access to the MDAnalysis core developer team to help shape future development of MDAnalysis
+* Networking opportunities to foster new collaborations
+* Community events, including a hackathon and social outings
+
```
Comment (micaela-matta) [inline _posts/2023-05-17-save-date-ugm2023.md:21]: mention hackathon

File: pages/ugm2023.md (hunk 3 above)
Comment (micaela-matta) [reply pages/ugm2023.md:34]: this is a "save the date" announcement. let's keep info to a minimum and add details about pricing, abstract submission and keynotes later after we've discussed internally one more time..

File: public/images/UGM2023_Logo.png
Comment (micaela-matta) [inline public/images/UGM2023_Logo.png:1]: suggestion to just have an image of Lisbon from https://unsplash.com/s/photos/lisbon on its own, for now.
I would like to have a banner with MDA font saying "MDAnalysis User Group Meeting"

**Review by micaela-matta — CHANGES_REQUESTED** [review]

File: pages/ugm2023.md (hunk 3 above)
Comment (jennaswa) [reply pages/ugm2023.md]: Done - I kept the headers for abstracts/keynotes as placeholders though

File: _posts/2023-05-17-save-date-ugm2023.md (hunk 4 above)
Comment (jennaswa) [reply _posts/2023-05-17-save-date-ugm2023.md:23]: Was included under 'Community events', but changed to make it more prominent

File: public/images/UGM2023_Logo.png
Comment (jennaswa) [reply public/images/UGM2023_Logo.png:1]: Agreed. @orbeckst - it looks like you uploaded the banner before (https://github.com/MDAnalysis/branding/tree/main/logos/banners). Would a UGM-specific banner be something you could easily generate?

**Review by micaela-matta — APPROVED** [review]

File: public/images/UGM2023_Logo.png
Comment (orbeckst) [reply public/images/UGM2023_Logo.png:1]: I'll make one.

Comment (orbeckst) [reply public/images/UGM2023_Logo.png:1]: @micaela-matta @jennaswa is this what you were thinking?

mdanalysis-banner-UGM-1280x144

Comment (orbeckst) [reply public/images/UGM2023_Logo.png:1]: See PR https://github.com/MDAnalysis/branding/pull/4

**Pull request merged**